### ISSUE TITLE
Rename "Add rules" and "Interesting"

### ISSUE THREAD

**Issue by kliegr** [KIZI/EasyMiner-BRE#18]
The current naming of links adding rules to clipboard/KB is confusing. Rules are added to clipboard with  "Add rules" and to knowledge base with "Interesting". Ideally, the links should say "Add to rule clipboard" and "Add to knowledge base".

**Comment by vojir** [comment]
Simmilar labels are placed in titles of the links.

**Comment by kliegr** [comment]
I do not agree -  What the user primarily sees is "Interesting". I would suggest reversing the labels, the button should say "Add to knowledge base", and the label "Interesting". Futhermore, there could be possibility to add all rules to KB, not just those on the first page.
